pages/pipelines/rules.md

```diff
@@ -0,0 +1,169 @@
+# Pipeline rules
+
+Pipeline rules is a Buildkite feature used to manage and organize permissions between pipelines and other resources.
+
+Rules express that an action is allowed between a source resource (eg. a pipeline) and a target resource (eg. another pipeline). Rules allow you to break out of the defaults provided by Buildkite such as cluster boundaries.
+
+## Pipeline rules best practices
+
+### How should I use rules?
+
+Use rules to manage permissions between pipelines and other resources. Rules are typically used in tandem with [clusters](/docs/clusters/overview) to increase security and control, where clusters set hard boundaries and rules provide exceptions.
```

Comment:
```suggestion
Use rules to manage permissions between pipelines and other resources. Rules are typically used in tandem with [clusters](/docs/clusters/overview) to increase security and control, where clusters set hard boundaries and rules define exceptions to those boundaries.
```

Reply:
It might be nice to add something like 'For example, a rule could be defined to allow a CI pipeline in one cluster to trigger a Deploy pipeline in another cluster.'

Reply:
Oh woops, I hadn't read all the way down. I see you've already got something like this in the Example rule use bit 😅